BITGROOM/configure.ac

```diff
@@ -0,0 +1,81 @@
+# Copyright by The HDF Group. All rights reserved.
+
+# This is the main configure file for the BITGROOM filter, a HDF5 plugin
+# library that enables BitGrooming quantization as an HDF5 filter.
+# This file is modified from a configure.ac from the hdf5_plugin project.
+
+# Allen Byrne, Ed Hartnett 1/14/19
+# Charlie Zender 9/10/20
+
+# Initialize autoconf.
+AC_PREREQ(2.59)
+AC_INIT(H5BGR, 1.0, nco-bugs@lists.sourceforge.net)
+AC_CONFIG_HEADER([config.h])
```

Comment:
config.h will need a prefix, "bitgroom_config.h"

Reply:
I won't change unless it's essential for cmake build.

Reply:
Yes, otherwise it will override other config.h files when building other files.